src/static/js/timeslider.js

```diff
@@ -71,90 +143,35 @@ function init() {
 
     // route the incoming messages
     socket.on('message', (message) => {
-      if (message.type == 'CLIENT_VARS') {
+      if (message.type === 'CLIENT_VARS') {
         handleClientVars(message);
       } else if (message.accessStatus) {
         $('body').html('<h2>You have no permission to access this pad</h2>');
-      } else if (message.type === 'CHANGESET_REQ') { changesetLoader.handleMessageFromServer(message); }
+      } else if (message.type === 'CHANGESET_REQ') {
+        changesetLoader.handleMessageFromServer(message);
+      }
     });
 
     // get all the export links
-    export_links = $('#export > .exportlink');
+    exportLinks = $('#export > .exportlink');
 
     $('button#forcereconnect').click(() => {
       window.location.reload();
     });
 
-    exports.socket = socket; // make the socket available
-    exports.BroadcastSlider = BroadcastSlider; // Make the slider available
+    // exports.socket = socket;
+    // exports.BroadcastSlider = BroadcastSlider;
```

Comment:
Please don't introduce commented-out code.

Reply:
Weird that I did...  I wonder why..  Thanks for spotting ...  Will investigate why/how this happened because I know not to.

Reply:
Oh...  This wasn't my PR :)  I'm an idjit 📦